### PR TITLE
Add branch to avoid m_pOmxBitstream dereferenced

### PR DESCRIPTION
**Pull request by LukeZheZhu**

### CODE REVIEW

File: omx_components/src/mfx_omx_vdec_component.cpp [hunk 1]
```diff
@@ -2013,7 +2013,8 @@ void MfxOmxVdecComponent::MainThread(void)
                         mfx_sts = DecodeFrame();
                     }
 
-                    m_pOmxBitstream->GetFrameConstructor()->Sync();
+                    if(m_pOmxBitsream)
+                        m_pOmxBitstream->GetFrameConstructor()->Sync();
```
Comment (TianmiChen) [inline omx_components/src/mfx_omx_vdec_component.cpp]: Is it possible to use the statement like this? 
if (m_pOmxBitstream && m_pOmxBitstream->GetFrameConstructor)
     m_pOmxBitstream->GetFrameConstructor()->Sync();

Comment (LukeZheZhu) [reply omx_components/src/mfx_omx_vdec_component.cpp]: > Is it possible to use the statement like this?
> if (m_pOmxBitstream && m_pOmxBitstream->GetFrameConstructor)
> m_pOmxBitstream->GetFrameConstructor()->Sync();

KW scan only reports m_pOmxBitstream dereferenced. Therefore, m_pOmxBitstream->GetFrameConstructor maybe unnecessary to check here.

Comment (TianmiChen) [reply omx_components/src/mfx_omx_vdec_component.cpp]: That's fine. Please send the PR to Zhiqiang and ask him to scan it.

Thanks,

Comment (DenWolf) [reply omx_components/src/mfx_omx_vdec_component.cpp]: There is common decoding process, in this place m_pOmxBitstream is always not null.
As result - this place of code is fully correct.

If you have met some potential problem by Code-Analysis tool for m_pOmxBitstream - it's better to remove unnecessary checking for this pointer.

Comment (dmitrybrazhkin) [reply omx_components/src/mfx_omx_vdec_component.cpp]: Agree with @DenWolf , this issue looks like false-positive from KW scan

**Pull request merged**